components/builder-protocol/protocols/originsrv.proto

```diff
@@ -177,7 +186,7 @@ message OriginMemberListResponse {
 
 message OriginMemberRemove {
   optional uint64 origin_id = 1;
-  optional uint64 user_id = 2;
+  optional string account_name = 2;
```

Comment:
Am I remembering this wrong or were we not supposed to change existing protocol fields?

Reply:
oh i should have mentioned. Nothing is using this message so this should be non breaking.